### PR TITLE
0.2.0 release

### PR DESCRIPTION
**Pull request by bengl**
Also fixing authorship on the TOMLs and fixing `--v8-help`

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -328,13 +328,19 @@ fn make_workers(config: Config) -> Result<Vec<Worker>, std::io::Error> {
...
 fn parse_args<'a>() -> (string::String, clap::ArgMatches<'a>) {
-    let (v8_flags_vec, args): (Vec<string::String>, Vec<string::String>) =
+    let (v8_flags_vec, mut args): (Vec<string::String>, Vec<string::String>) =
         std::env::args().partition(|arg| arg.starts_with("--v8-"));
     let v8_flags_vec: Vec<string::String> = v8_flags_vec
         .iter()
         .map(|f| f.replace("--v8-", "--"))
         .collect();
     let v8_flags = v8_flags_vec.join(" ");
+    if v8_flags.contains("--help") {
```
Comment (tlhunter) [inline src/main.rs]: Shouldn't this be "--v8-help"?

Comment (tlhunter) [reply src/main.rs]: nm, it's already been stripped

Comment (bengl) [reply src/main.rs]: Nope. By this point it has already filtered out the `v8-` prefix we add to all the v8 flags.

**Comment by bengl** [comment]
To be clear: GitHub is showing these commits out of order. See [my branch](https://github.com/bengl/osgood/commits/bengl/0.2.0-release) for the correct order.

File: src/main.rs [hunk 2]
```diff
@@ -328,13 +328,19 @@ fn make_workers(config: Config) -> Result<Vec<Worker>, std::io::Error> {
...
+        // this suppresses error asking for a filename if just trying to get --v8-help, while stil
+        // requiring it in all other cases
+        args.push("APPFILE".to_owned());
```
Comment (tlhunter) [inline src/main.rs]: bit of a hack, but I suppose it's fine

Comment (tlhunter) [reply src/main.rs]: Though, would it make more sense to set the value to `IGNORED_FILENAME_FOR_HELP_PURPOSES` or something like that? Might make someone's debugging experience nicer someday.

Comment (bengl) [reply src/main.rs]: sure i'll change that.

**Pull request merged**